pyproject.toml

```diff
@@ -15,13 +15,16 @@ terralab = "terralab.cli:cli"
 [tool.poetry.dependencies]
 python = "^3.12"
 terra-scientific-pipelines-service-api-client = "0.1.7"
+# teaspoons-client = {path = "/Users/marymorg/workbench/terra-scientific-pipelines-service/python-client/generated"}
```

Comment:
should we continue to remove this before merging or just leave it in here (same with the localhost api url in the config)